src/components/activities/manageBillActivity/styles.scss

```diff
@@ -0,0 +1,259 @@
+          font-size: 1rem;
+          top: 3px;
+        }
+      }
+
+      .accordion_details {
+        &::-webkit-scrollbar {
+          width: 5px;
+        }
+
+        &::-webkit-scrollbar-track {
+          box-shadow: inset 0 0 5px grey;
+          border-radius: 10px;
+        }
+
+        &::-webkit-scrollbar-thumb {
+          background: darken($c-gray, 25%);
+          border-radius: 10px;
+        }
+
+        &::-webkit-scrollbar-thumb:hover {
+          background: darken($c-gray, 10%);
+        }
+      }
+      & + .accordion {
+        border-top: 1px solid $c-grey-medium;
+        //border-top: 1px $c-grey-medium;
+      }
+    }
+
+    @include susy-media($smartphone) {
+      overflow: hidden;
+      max-height: 0px;
+      transition: 0.5s max-height ease-in-out;
+    }
+  }
+
+  .searchBills__formData__trigger_mobile {
+    display: none;
+    color: $c-white;
+    position: relative;
+    padding: 15px 40px 15px 15px;
+    @include susy-media($smartphone) {
+      display: block;
+    }
+    svg.arrow_icon {
+      position: absolute;
+      top: 12px;
+      width: 30px;
+      right: 10px;
+      fill: $c-white;
+      transition: 0.3s transform ease-in-out;
+    }
+  }
+
+  .searchBills__profilePictureContainer_wrapper {
+    display: flex;
+    padding: 20px;
+    margin: 20px;
+    align-items: center;
+    border-radius: 10px;
+    background-color: lighten($c-gray, 10%);
+  }
+
+  .searchBills__profilePictureContainer {
+    width: 60px;
+    height: 60px;
+    display: inline-block;
+    .profilePicture_mask {
+      width: 60px;
+      height: 60px;
+    }
+  }
+
+  .searchBills__header__info {
+    margin-left: 12px;
+    .searchBills__header__info__item {
+      color: $c-white;
+      display: block;
+      text-transform: uppercase;
+    }
+  }
+
+  .searchBills__formData__item {
+    width: 100%;
+    padding: 7px 20px;
+    color: $c-white;
+    display: inline-block;
+
+    .searchBills__formData__item__label {
+      font-size: 0.85rem;
+    }
+
+    &:last-child {
+      margin-bottom: 10px;
+    }
+
+    &.longText {
+      .searchBills__formData__item__label {
+        font-weight: bold;
+        margin-bottom: 10px;
+      }
+      .searchBills__formData__item__value {
+        font-weight: initial;
+        font-size: smaller;
+      }
+    }
+  }
+
+  .searchBills__formData__item__value {
+    font-weight: bold;
+  }
+
+  .searchBills__content {
+    flex: 3;
+    display: flex;
+    flex-direction: column;
+    @include susy-media($smartphone) {
+      display: block;
+      flex: initial;
+      margin-top: 35px;
+      border-radius: 10px;
+      padding: 40px 25px 25px;
+      flex-direction: initial;
+      background-color: $c-white;
+      box-shadow: 0 4px 8px 0 rgba(48, 49, 51, 0.1);
+    }
+  }
+
+  .searchBills__content__header {
+    background-color: #f5f5f5;
+  }
+
+  .footer {
+    position: relative;
+  }
+    
+  .active {
+    background-color: lighten($c-gray, 10);
+  }
+
```

Comment:
There is some padding that we miss inside the accordion:

![image](https://user-images.githubusercontent.com/8697550/135483817-dee4804b-efa3-43dc-8c19-4a9599ee4cad.png)

Reply:
OK